.travis.yml

```diff
@@ -109,134 +19,30 @@ matrix:
       install:
         - rustup target add aarch64-apple-ios
       script:
-        # Differs from standard script: includes aarch64-apple-ios cross-build
-        - cargo test --tests --no-default-features
-        # TODO: add simd_support feature:
-        - cargo test --features=serde1,log
-        - cargo test --examples
-        - cargo test --manifest-path rand_core/Cargo.toml
-        - cargo test --manifest-path rand_core/Cargo.toml --no-default-features
-        - cargo test --manifest-path rand_distr/Cargo.toml
-        - cargo test --manifest-path rand_isaac/Cargo.toml --features=serde1
-        - cargo test --manifest-path rand_pcg/Cargo.toml --features=serde1
-        - cargo test --manifest-path rand_xorshift/Cargo.toml --features=serde1
-        - cargo test --manifest-path rand_xoshiro/Cargo.toml
-        - cargo test --manifest-path rand_chacha/Cargo.toml
-        - cargo test --manifest-path rand_hc/Cargo.toml
-        - cargo test --manifest-path rand_jitter/Cargo.toml
-        - cargo test --manifest-path rand_os/Cargo.toml
+        - bash utils/ci/script.sh
```

Comment:
Do we really need macOS and iOS tests?

Reply:
I don't know; possibly not. These aren't too slow in any case (unlike the WASM tests).